### ISSUE TITLE
Icon component should have method of specifying alt text

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#1857]
While attempting to create a `c4_icon` shortcode for our new WordPress theme, I realized I couldn't actually pass anything to the SVG template's `before` block because it would have to bust through the icon component first. Ideally the component could have a simplified way of specifying `alt` text so this wouldn't be necessary.
